### PR TITLE
DOCS: Added dependencies to linux compilation instructions for fedora.

### PR DESCRIPTION
**Pull request by ciscon**

### CODE REVIEW

File: sbar.c [hunk 1]
```diff
@@ -53,6 +53,7 @@ cvar_t  sbar_drawsigils     = {"scr_sbar_drawsigils", "1"};
 cvar_t  sbar_drawhealth     = {"scr_sbar_drawhealth", "1"};;
 cvar_t  sbar_drawarmor      = {"scr_sbar_drawarmor",  "1"};;
 cvar_t  sbar_drawammo       = {"scr_sbar_drawammo",   "1"};;
+cvar_t  sbar_lowammo       = {"scr_sbar_lowammo",   "5"};;
```
Comment (jite) [inline sbar.c]: Some small notes, please correct nbr of spaces for alignment (not tabs if used, on phone so only see its not aligned) :)

Comment (jite) [reply sbar.c]: Oh and feel free to remove the extra unnecessary semicolons on those rows

File: sbar.c [hunk 2]
```diff
@@ -998,7 +1000,7 @@ static void Sbar_DrawCompact_WithIcons(void) {
 	align = scr_compactHudAlign.value ? 1 : 0;
 	for (i = 0; i < 4; i++) {
 		snprintf(str, sizeof(str), "%d", cl.stats[STAT_SHELLS + i]);
-		if (cl.stats[STAT_SHELLS + i] < 5)
+		if (cl.stats[STAT_SHELLS + i] <= sbar_lowammo.value)
```
Comment (jite) [inline sbar.c]: Use .integer instead

File: sbar.c [hunk 3]
```diff
@@ -1035,7 +1037,7 @@ static void Sbar_DrawCompact(void) {
...
```
Comment (jite) [inline sbar.c]: Same

File: sbar.c [hunk 4]
```diff
@@ -1069,7 +1071,7 @@ static void Sbar_DrawCompact_TF(void) {
 	Sbar_DrawNum (86, 0, cl.stats[STAT_HEALTH], 3, cl.stats[STAT_HEALTH] <= 25);
 	for (i = 0; i < 4; i++) {
 		snprintf(str, sizeof(str), "%d", cl.stats[STAT_SHELLS + i]);
-		if (cl.stats[STAT_SHELLS + i] < 5)
+		if (cl.stats[STAT_SHELLS + i] <= sbar_lowammo.value)
```
Comment (jite) [inline sbar.c]: Same

File: COMPILING_ON_LINUX.txt [hunk 5]
```diff
@@ -10,7 +10,7 @@ HOW TO BUILD ON UBUNTU:
  $ sudo apt install git build-essential libsdl2-2.0-0 libsdl2-dev libjansson-dev libexpat1-dev libcurl4-openssl-dev libpng-dev libjpeg-dev libspeex-dev libspeexdsp-dev
 
 - For Fedora 25+:
- $ sudo dnf install mesa-libGL-devel SDL2-devel make gcc jansson-devel expat-devel libcurl-devel libpng-devel libjpeg-turbo-devel speex-devel speexdsp-devel
+ $ sudo dnf install vim-common pcre-devel mesa-libGL-devel SDL2-devel make gcc jansson-devel expat-devel libcurl-devel libpng-devel libjpeg-turbo-devel speex-devel speexdsp-devel
```
Comment (javian) [inline COMPILING_ON_LINUX.txt]: Why is vim-common required ? It sounds a bit weird.

Comment (jite) [reply COMPILING_ON_LINUX.txt]: It provides 'xxd' which we currently use for json -> C arrays at compile time.

Comment (javian) [reply COMPILING_ON_LINUX.txt]: Cool, didn't know that.

**Pull request merged**